traffic_ops/app/lib/API/User.pm

```diff
@@ -277,6 +285,7 @@ sub current {
 			@data, {
 				"id"              => "0",
 				"username"        => $current_username,
+				"tenantId"	  => $self->current_user_tenant(),
```

Comment:
currently we have 2 ways to login:

1. checking ldap credentials
2. checking username/password in the tm_user table

notice that this is in the isLDAP() scenario so we give role=read-only to the user. we should also probably give tenantId = (the id of the root tenant)

does that make sense? so really, this would be fine:

"tenantId"	  => 1,

if we can always guarantee that the root tenant has ID=1